README.md

```diff
@@ -515,14 +515,64 @@ also emitted from the client via a `metrics` event:
 ```
 
 #### client.on('after', function(req, res, err) {... });
-StringClient and JsonClient emit an `after` event after each completed request
+The client will emit an `after` event after each completed request
 (successful or not). This can be useful for attaching after handlers to report
 metrics or other useful information about the client. When using this event,
 the timings and metrics information is available via the request getters,
-`req.getTimings()` and `req.getMetrics()`. Note that the low level HttpClient
-does not support this method, as that client is a thin wrapper over the http
-streams. As such, the user is in full control of the request lifecycle, and
-`after` would not have a clear meaning.
+`req.getTimings()` and `req.getMetrics()`.
+
+As the low level HttpClient's callback interface exposes the streams
+directly to the consumer, there are a few caveats around using `after` with the
+HttpClient. Consumers must consume the response stream (by consuming the `data`
+event on the response) in order for the `after` event to be fired. For example,
+imagine an HttpClient request returning a 500:
```

Comment:
Should this be "returning a 200"?

Reply:
nit: The word "imagine" seems to suggest that we'll mention something about this client after this sample code, but instead we switch to covering clients with errors.

Maybe instead of "imagine a HttpClient request returning a 200", maybe phrase it as "For example, here's how a HttpClient instance consumes the response's data to trigger the `'after'` event:", or something along those lines?